addons/web/static/src/views/fields/properties/property_definition.scss

```diff
@@ -6,21 +6,9 @@
     max-height: 70vh;
     overflow-y: auto;
 
-    table {
-        width: 500px;
-    }
-
-    .o_td_label {
-        width: 120px;
-    }
-
-    .o_field_property_definition_header {
-        max-width: 500px;
-
-        input {
-            font-size: 1.2rem;
-            border-width: 1px;
-        }
+    .o_modal_container {
+        min-width: 425px;
+        max-width: 550px;
```

Comment:
Arbitrary and veeeeery specific 😅
Isn't there something we can refer too rather than force a specific value? Normally you should ask yourself: "would this value still be correct if we change the base font-size?" or grid-gutter? or the base spacing?"